.circleci/config.yml

```diff
@@ -1,7 +1,321 @@
+              W1tm7qbAOm1Iv4uNYVL1qgpq1GnD5qpWSACGsVSE3OGELlNaVz8fgVdz6zT+rU2A
+              tp2t795UlrvaLgFI4wITqJF3LoTfy2MZu8JYCzlKM5pZksmEmJfR0RDAot2grtD3
+              H5A+PZfUIZ/8BhmdaOAv5i647unfVF6UpPYejZ0rb67oEazxdeIHK3aD5AjurdsO
+              UpW/PMwsbaltp4iI7hvUfRX7Afb5fPXIhv9pHh1xWYl3djUNWmFoiMMP4tuxpOBo
+              y+T4maQaiDSHAoIBADrlZ9EIMclMnNXJYE4O4fbFesUvV0lHM3+ayQgXiH0Vg5Nl
+              2xjPlqBX0bDajVluPU6AF3GYxfoSLv1GXqTvb9iVpKXrAHp+nef0uxMP9ltZT6Qz
+              UA1wh3x2OBFJ0hK0B1FsmeSHS8VDQye615jEA8iMM/GrbnnM/p7ccEcOkyO8YJSj
+              I/rNbzN6u8yAPZCzyx6Hy4w/xsdf1acslOHJj3kyX/cwqCGxnc/GvVR2OSZyHVnT
+              sLnGj7NEeudwvKlyxuzj5CMmz111wVEI2olgQa9Sl+EBu140mnDNTNYCA7OnwE3z
+              GoFMOrXC2mf2ZfSge4orbL5Nellnt51pOLp2x8ECggEBALM8Mazw/FOF9mbdgjJM
+              PFGSaa7rBcVJwdHttDHBmlPI6wzsvFEMPru6nfx76KJQbORqK9r13sN5fyzof59m
+              TwsbMt/cFSnOQJ39M7YPstDofbl20cDOduUzpEVsRvVKokhqGB3XVRiuZ1y+8WSz
+              Wh7OiTu3AwzKsrcYXkZQdnlRBq0iYcfLPKzHqUJLLzbOH9Q6djL5c8V/qLNfvNI1
+              2HqKVqV8Ex+zKJhBWRAe+x3bKnbS7MPQ6zNfsOdgCmhydwRCquPzpr7JU/PFZh+4
+              b31cHgFrIZR2d2AzW1XcSLzsqa2vUs2RKOIu2deAPaUI/66zCZeTnGBNEFza76Ga
+              1oUCggEAA38oXcnputwL103SeD8+uwHjtTf183Rucr+Ryqz6GymiWjlzELqu7TRd
+              yadAaNg9CuXmYS33Jtk/UNS0k9FvYqGTR+SBXIZr6nt9ZFd0SNlQkwkAQCsuekEs
+              nJlmUZax7DxXMgIHMKDboHZYM/MhgzEGSALmhU5LZ76MS17v3NEPxYpVHxjAotxW
+              g03HjWTltS8Bgt6u0KFTGJKEUcfwvWKZtjk5Fc1heZ49zh1nU3zo9C/h8iiijTy2
+              s/YksP6cxveae4b7soN4rD/vnfsmKcG+DnTf6B8Zbm6tI2TneYOfFSCryp+yDnaJ
+              PIDNiTxNecePOmrD+1ivAEXcoL+e1w==
+              -----END PRIVATE KEY-----
+    authorizations:
+        - id: alice
+          key: fs5wgcer9qj819kfptdlp8gm227ewxnzvsuj9ztycsx08hfhzu
+          signers:
+              - webextensions-rsa
+        - id: bob
+          key: 9vh6bhlc10y63ow2k4zke7k0c3l9hpr8mo96p92jmbfqngs9e7d
+          signers:
+              - webextensions-rsa-with-recommendation
+    ###########################################################################
+    #
+    #               The autograph configuration ends here.
+    #
+    ###########################################################################
+
+  defaults: &defaults
+    working_directory: ~/addons-server
+    docker:
+      # This is the python version we run in production.
+      - image: circleci/python:3.8
+      # Below are services this project depends on. In addition to these
+      # services, we also need autograph, which is started in the `test` job
+      # because we need to pass a configuration file to it and it's not
+      # possible in this section.
+      #
+      # Most settings below should be kept in sync with `docker-compose.yml`.
+      - image: redis:2.8
+      - image: memcached:1.4
+      - image: mysql:8.0
+        # This is needed to avoid an error with an authentication plugin.
+        command: --default-authentication-plugin=mysql_native_password
```

Comment:
Mmm that is surprising. Are we using the right MySQL client libs in CircleCI ?

Reply:
I am not sure. It's using whatever is the default on Ubuntu I guess.

Reply:
That's a problem, we specifically updated the MySQL client libs to use MySQL's in our image. We were also installing them in travis (See `before_install` first line).

Reply:
fixed